srcpkgs/lsd/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'lsd'
+pkgname=lsd
+version=0.16.0
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+#makedepends="rust cargo"
+depends="awesome-terminal-fonts"
```

Comment:
This package does not exist, maybe you wanted `font-awesome` or `font-awesome5`?